docs/index.rst

```diff
@@ -61,6 +61,14 @@ Here is a simple example of how it looks in practice: ::
    :alt: animation
    :align: center
 
+To convert the plot to a 2D variant, at the following line before `scene.plot()`: ::
+
+   scene.convert_to_2d()
```

Comment:
You seem to have forgotten to update this line.

If you prefer or know a clearer name than `set_plot_as_2d`, feel free to do so. Some options I could think of are:
- `as_2d_projection`
- `to_2d_projection`
- `as_2d_perspective`
- `to_2d_perspective`
- `as_2d_view`
- `to_2d_view`

Reply:
thanks for noticing! 

Since this feature is not "real" 2d but an orthogonal projection of a 3D situation. Would you like the name `as_orthogonal_projection_plot`?

Reply:
I'm fine with most names. The advantage of having `2d` in the name is that it is easier to find the method from a user's perspective. Personally, I think that the word `perspective` covers that you are not actually projecting the objects to 2D, but `as_orthogonal_projection` would indeed have the advantage of more accurately describing what the method does internally.